code/agents.py

```diff
@@ -67,6 +67,8 @@ def calculate_fish_forces(self):
         for (fish, pos) in neighbours:
 
             magnitude = k / np.linalg.norm(pos - self.position)**power
+            if magnitude > 2:
```

Comment:
no its capped. In case fishes start on top of eachother the force will be so big that it pushes them apart very far vary fast